libs/galley-types/src/Galley/Types/Teams.hs

```diff
@@ -590,6 +606,8 @@ parseTeamMember = withObject "team-member" $ \o ->
     TeamMember <$> o .:  "user"
                <*> o .:  "permissions"
                <*> parseInvited o
+               -- Default to disabled if missing
```

Comment:
Is that a `Monoid` with overwrite semantics?  Probably, but even more probably that's over-thinking things..

Reply:
Do you mean the type itself or the JSON parser?

I'm just using built-in Aeson combinators for specifying a default 🤷‍♂

Reply:
i meant the type.  If there was a monoid instance then we could say `mempty` here, which takes semantics away from the serialization code.

(not important at all.)